apps/playwright/tests/admin/location-preference-validation.spec.ts

```diff
@@ -85,64 +83,90 @@ test.describe("Location Preference Validation", () => {
         },
       },
     });
-
-    const createdPool = await createAndPublishPool(adminCtx, {
-      userId: createdUser?.id ?? "",
-      skillIds: skill ? [skill?.id] : undefined,
-      name: {
-        en: `App location preference ${testId} (EN)`,
-        fr: `App location preference ${testId} (FR)`,
-      },
-    });
-
-    const applicantCtx = await graphql.newContext(
-      createdUser?.authInfo?.sub ?? "applicant@test.com",
-    );
-    const applicant = await me(applicantCtx, {});
-
-    const candidate = await createAndSubmitApplication(applicantCtx, {
-      userId: applicant.id,
-      poolId: createdPool.id,
-      personalExperienceId: applicant?.experiences?.[0]?.id ?? "",
-      signature: `${applicant.firstName} signature`,
-    });
-
-    application = candidate;
     user = createdUser ?? { id: "" };
-    id = createdPool.id;
   });
 
   test.afterAll(async () => {
     if (user) {
       await deleteUser(adminCtx, { id: user.id });
-      await deletePool(adminCtx, { id: id });
     }
   });
 
-  test("Work Location preference update in Admin view", async ({ appPage }) => {
+  test("Location preference update in Users table", async ({ appPage }) => {
     locationPrefPage = new LocationPreferenceUpdatePage(appPage.page);
     const userName = user?.firstName ?? "";
     await loginBySub(appPage.page, testConfig.signInSubs.adminSignIn, false);
     userPage = new UserPage(appPage.page);
-    await userPage.goToIndex();
-    await userPage.searchUserByName(userName, "Candidate name");
-    await appPage.page.locator(`a:has-text("${userName} User")`).click();
-    await appPage.waitForGraphqlResponse("AdminApplicantProfilePage");
-    await expect(
-      appPage.page.getByRole("heading", { name: userName }),
-    ).toBeVisible();
+    await userPage.goToUserProfile(user.id);
     await appPage.page
       .getByRole("button", { name: "Work preferences", exact: true })
       .click();
     await locationPrefPage.validateSelectedFlexWorkLocOptions();
+    await userPage.goToIndex();
+    await locationPrefPage.showOrHideColumns();
+    await locationPrefPage.filterFlexWorkLocation(
+      [FlexibleWorkLocation.Hybrid, FlexibleWorkLocation.Onsite],
+      [WorkRegion.Atlantic],
+    );
+    await appPage.waitForGraphqlResponse("UsersPaginated");
+    await userPage.searchUserByName(userName, "Candidate name");
+    await expect(
+      appPage.page.getByRole("columnheader", {
+        name: /Flexible work location options/i,
+      }),
+    ).toHaveAccessibleName(/Flexible work location options/i);
+    await locationPrefPage.verifyFlexibleWorkLocationData(userName);
+    await expect(
+      appPage.page.locator(`a:has-text("${userName} User")`),
```

Comment:
Could this be `getByRole("link", { name: RegEx(`${userName} User`, "i" }))`.

Or is it different than that? I'm not very experienced with raw locators like this :sweat_smile: